libraries/AP_Motors/AP_MotorsMulticopter.cpp

```diff
@@ -329,7 +329,7 @@ void AP_MotorsMulticopter::update_throttle_filter()
     }
 
     // calculate slope normalized from per-micro
-    const float rate = fabsf(_throttle_slew.slope() * 1e6);
+    const float rate = fabsf(_throttle_slew.slope() * 1e6f);
```

Comment:
this implies we're not setting -fsingle-precision-constant in the build, which would be a real bug, as it means different behaviour that STM32

Reply:
It was indeed a real bug. Fixed now.